extra/dynflow-executor.example

```diff
@@ -0,0 +1,77 @@
+# Author: Riley Shott
+# https://gist.github.com/Ginja/fb123759d48b04b5f338
+# dynflow-executor - starts or stops the Dynflow executor for Foreman tasks
+#
+# chkconfig:   345 99 20
+# description: dynflow-executor is a rake task that comes with \
+#              the foreman-tasks gem
+# processname: dynflow-executor
+# pidfile:     /usr/local/foreman/tmp/pids/dynflow_executor_monitor.pid
+# pidfile:     /usr/local/foreman/tmp/pids/dynflow_executor.pid
+
+# Source function library.
+. /etc/rc.d/init.d/functions
+
+foreman_user='foreman'
+rvm='/usr/local/.rvm/bin/rvm'
+gemset='ruby-2.1.5@foreman-1.9.1'
+foreman_app_path='/usr/local/foreman'
+dynflow_executor="$(cd /usr/local/foreman && /usr/local/.rvm/bin/rvm ruby-2.1.5@foreman-1.9.1 do bundle show foreman-tasks)/bin/dynflow-executor"
```

Comment:
Why not use the `foreman_app_path`, `rvm` and `gemset` variables instead of explicit `/usr/local/foreman`, `/usr/local/.rvm/bin/rvm` and `ruby-2.1.5@foreman-1-9-1`?